release/models/nat/openconfig-nat.yang

```diff
@@ -0,0 +1,1008 @@
+module openconfig-nat {
+
+  yang-version "1";
+
+  // namespace
+  namespace "http://openconfig.net/yang/nat";
+
+  prefix "oc-nat";
+
+  // import some basic types
+  import ietf-inet-types { prefix inet; }
+  import ietf-yang-types { prefix yang; }
+  import openconfig-interfaces { prefix oc-if; }
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-acl { prefix oc-acl; }
+
+  // meta
+  organization "OpenConfig working group";
+
+  contact
+    "OpenConfig working group
+    www.openconfig.net";
+
+  description
+    "This module defines configuration and operational state data
+    for Network Address Translation (NAT) functionality.
+
+    NAT enables translation of IP addresses and/or ports between
+    different network domains.
+
+    The model supports:
+    - Source NAT (SNAT) for outbound traffic translation
+    - Destination NAT (DNAT) for inbound traffic translation
+    - Static and dynamic address/port mappings
+    - Policy-based NAT using ACLs
+    - Active translation table showing current NAT entries
+    - Multiple address families (IPv4/NAT44, IPv6/NAT66).";
+
+  oc-ext:openconfig-version "0.1.0";
+
+  revision "2025-07-15" {
+    description
+      "Initial revision";
+    reference "0.1.0";
+  }
+
+  // Identity statements
+
+  identity NAT_TYPE {
+    description
+      "Base identity for NAT types.";
+  }
+
+  identity SOURCE {
+    base NAT_TYPE;
+    description
+      "Source NAT - translates source addresses/ports of outbound
+      traffic.";
+  }
+
+  identity DESTINATION {
+    base NAT_TYPE;
+    description
+      "Destination NAT - translates destination addresses/ports
+      of inbound traffic.";
+  }
+
+  identity ADDRESS_FAMILY {
+    description
+      "Base identity for NAT address families.";
+  }
+
+  identity IPV4 {
+    base ADDRESS_FAMILY;
+    description
+      "IPv4 address family for NAT44 translations.";
+  }
+
+  identity IPV6 {
```

Comment:
Since this model originally started off as exclusively NAT44, is there a need for the IPv6 case? If not, lets defer adding this identity until it is needed

Reply:
I can remove the 

    leaf prefix-length {
      type uint8 {
        range "1..128";
      }
      description
        "Prefix length for the address pool. For IPv4, valid range
        is 1-32. For IPv6, valid range is 1-128.";
    }